static/js/settings_account.js

```diff
@@ -104,6 +104,36 @@ exports.set_up = function () {
         }
     });
 
+    function send_pass_reset() {
+        var email = $('#email_value').text().trim();
+        var form_data = new FormData();
+
+        form_data.append("email", email);
+        channel.post({
+            url: '/accounts/password/reset/',
+            data: form_data,
+            cache: false,
+            processData: false,
+            contentType: false,
+        });
+    }
+
+    $('#forgot_password').on('click', function () {
+        $('#forgot_password').hide();
+        $('#reset_password').show();
+    });
+
+    $('#reset_password').on('click', function () {
+        $('#reset_password').hide();
+        send_pass_reset();
+        $('#reset_sent').show();
```

Comment:
This version cheerfully says "Check your email!" the moment the user clicks the link. The user may be skeptical of that, and they'd be right -- because we actually aren't waiting for any kind of reply from the server! We could have completely lost our connection to the server, and we'd still instantly show this confirmation.

Try it -- open up this page, stop your dev server completely, and then hit "Reset password".

We should show a little spinner here until we get a successful reply. Similarly, on "Resend" we should show a spinner.